hr_employee_calendar_planning/migrations/12.0.1.1.0/post-migration.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE resource_calendar
+        SET auto_generate = true
+        WHERE active = false""",
```

Comment:
Should we ensure it better filtering those with the name like   'Auto generated calendar for employee'

Reply:
It can't be due to the translations.